app/routes/organen/orgaan/mandatarissen.js

```diff
@@ -74,6 +76,8 @@ export default class OrganenMandatarissenRoute extends Route {
       selectedBestuursperiode: parentModel.selectedBestuursperiode,
       mandatarisNewForm: mandatarisNewForm,
       legislatuurInBehandeling: isDistrict ? false : legislatuurInBehandeling,
+      isPublicationStatusEditable:
+        !bestuurseenheid.isOCMW || (bestuurseenheid.isOCMW && isBcsdOrgaan),
```

Comment:
Wouldn't this just do the same?
```suggestion
      isPublicationStatusEditable:
        isBcsdOrgaan || !bestuurseenheid.isOCMW,
```